samples/cpp/prompt_lookup_decoding_lm/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2023-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+if(TARGET openvino_tokenizers)
+    set(OPENVINO_TOKENIZERS_PATH $<TARGET_FILE:openvino_tokenizers>)
+else()
+    message(FATAL_ERROR "multinomial_causal_lm must be compiled as part of OpenVIINOGenAI to have the path to openvino_tokenizers hardcoded.")
```

Comment:
@Wovchena I suppose even as part of OpenVINO GenAI package path to `openvino_tokenizers` library can be deduced. 
E.g. something like `${CMAKE_CURRENT_SOURCE_DIR}/../../../runtime/bin/Release/`